test/conviction-voting.js

```diff
@@ -951,14 +986,52 @@ contract('ConvictionVoting', ([appManager, user, beneficiary]) => {
         })
       })
     })
+
+    context('addProposal(title, link, requestedAmount, stableRequestAmount (true), beneficiary)', () => {
+
+      const stableRequestedAmount = 1000
+      let proposalId, actionId
+
+      beforeEach(async () => {
+        const addProposalReceipt = await convictionVoting.addProposal('Proposal 1', '0x', stableRequestedAmount, true, beneficiary)
+        proposalId = getEventArgument(addProposalReceipt, 'ProposalAdded', 'id')
+        actionId = getEventArgument(addProposalReceipt, 'ProposalAdded', 'actionId')
+      })
+
+      context('stakeToProposal(proposalId, amount)', () => {
+        const stakeAmount = 1000
+
+        beforeEach(async () => {
+          await convictionVoting.stakeToProposal(proposalId, stakeAmount)
+        })
+
+        context('executeProposal(proposalId)', () => {
+          it('converts stable amount to request token and transfers to beneficiary', async () => {
+            const vaultBalanceBefore = await requestToken.balanceOf(vault.address)
+            const beneficiaryBalanceBefore = await requestToken.balanceOf(beneficiary)
+            const requestTokenReturnAmount = stableRequestedAmount / FEE_TOKEN_PRICE_IN_STABLE_TOKEN
+            const stakeAmount = 10000
+            await convictionVoting.stakeToProposal(proposalId, stakeAmount)
+            await convictionVoting.mockAdvanceBlocks(40)
+
+            await convictionVoting.executeProposal(proposalId)
+
+            const vaultBalanceAfter = await requestToken.balanceOf(vault.address)
+            assert.equal(vaultBalanceAfter, vaultBalanceBefore - requestTokenReturnAmount, 'Incorrect vault balance')
+            const beneficiaryBalanceAfter = await requestToken.balanceOf(beneficiary)
+            assert.equal(beneficiaryBalanceAfter.toNumber(), beneficiaryBalanceBefore.toNumber() + requestTokenReturnAmount, 'Incorrect beneficiary balance')
+          })
+        })
+      })
+    })
   })
 
-  context('Pure functions', () => {
+  context.only('Pure functions', () => {
```

Comment:
Might forgot to remove 😄 
```suggestion
  context('Pure functions', () => {
```

Reply:
Never mind, just saw it updated on next PR